cli/tests/test_observability_v8_upgrade_continuity_checker.py

```
@@ -0,0 +1,160 @@
+
+from __future__ import annotations
+
+import importlib.util
+from pathlib import Path
+
+import pytest
+
+ROOT = Path(__file__).resolve().parents[2]
+CHECKER_PATH = ROOT / "scripts" / "check_observability_v8_upgrade_continuity.py"
+SPEC = importlib.util.spec_from_file_location("upgrade_continuity_checker", CHECKER_PATH)
+assert SPEC is not None and SPEC.loader is not None
```

Ruff (0.15.21)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)